packets1/connect.go

```diff
@@ -77,7 +77,7 @@ func (p *Connect) Unpack(buf []byte) error {
 	return nil
 }
 
-func (p Connect) String() string {
+func (p *Connect) String() string {
```

Comment:
This is incorrect. `Stringer` interface implementation _must_ have a non-pointer receiver.